### DISCUSSION TITLE
Flask Saying 404 when I deploy

### DISCUSSION THREAD

**Discussion by sr5434** [vercel/community#3596] [Help]
### Summary

I have a [flask app](https://github.com/sr5434/personal-assistant) that I am trying to deploy. When I do, the app breaks. How do I fix this?

### Steps to Reproduce

Run vercel dev on your local system to see for yourself. The flask code is in the app.py file.

**Comment by w3bdesign** [comment]
@sr5434 

You should try moving the Python files to a new folder called /api

**Reply by sr5434** [reply]
I tried but that did not work. I just ended up moving my app to render.io

**Reply by spielbude-ev** [reply]
does vercel support docker? [their guides](https://vercel.com/guides/does-vercel-support-docker-deployments) suggest that they do not.

**Comment by spielbude-ev** [comment]
I have a similar problem -- and even the sample flask application template from vercel is (currently?) not working but only giving a 404 error.
